src/Tribe/Main.php

```diff
@@ -40,12 +35,53 @@ public static function instance() {
 	 */
 	protected function __construct() {
 
+		$this->init();
 
 	}
 
 	public function init() {
 
+		// Setup bootstrap on earliest hook available to the extension.
+		add_action( 'tribe_plugins_loaded', [ $this, 'bootstrap' ], 11 );
+
+		$this->get_settings();
+	}
+
+	/**
+	 * Bootstrap Plugin
+	 *
+	 * @since TBD
+	 *
+	 */
+	public function bootstrap() {
+
+		// Intialize the Service Provider for Event Tickets HubSpot Integration.
+		tribe_register_provider( Service_Provider::class );
+
+	}
+
+	/**
```

Comment:
If we call `tribe( 'tickets.hubspot.admin.settings' )` then we don't need this method

Reply:
removed it!